setup.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Without this the following error occurs when I run tox:

  SyntaxError: Non-ASCII character '\xc3' in file setup.py on line 14, but
  no encoding declared; see http://python.org/dev/peps/pep-0263/ for
  details

Triggered, I believe, by the umlaut in your name.